components/aws/sagemaker/tests/unit_tests/README.md

```diff
@@ -0,0 +1,43 @@
+# Unit tests for AWS SageMaker KFP Components 
+
+These unit tests test the user input parsing part of the code.  
```

Comment:
Either can we add more information here or simply removing this explanation.

Reply:
removed